docs/sources/explore/index.md

```diff
@@ -145,7 +145,7 @@ Suggestions can appear under the query field - click on them to update your quer
 
 ### Table filters
 
-Click on the filter button <span title="Filter for label" class="logs-label__icon fa fa-search-plus"></span> in a labels column in the Table panel to add filters to the query expression. This works with multiple queries too - the filter will be added for all the queries.
```

Comment:
This was only place in docs where we've had icon. Maybe instead of having an icon, we could improve documentation.